OpenRA.Game/Settings.cs

```diff
@@ -20,26 +20,66 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchArgs
```

Comment:
Is it me, or are these duplicating the `openra://` protocol and `.orarep` OS integration?